radio/src/translations/se.h

```diff
@@ -655,33 +674,38 @@
 #define TR_VOLTAGE                      INDENT"Volt"
 #define TR_CURRENT                      INDENT"Ampere"
 #define TR_SELECT_MODEL                 "Välj modell"
+#define TR_MODEL_CATEGORIES             "Modellkategori"
 #define TR_MODELS                       "Modeller"
-#define TR_SELECT_MODE                  "Select mode"
+#define TR_SELECT_MODE                  "Välj läge"
+#define TR_CREATE_CATEGORY              "Skapa kategori"
+#define TR_RENAME_CATEGORY              "Byt kategorinamn"
+#define TR_DELETE_CATEGORY              "Ta bort kategori"
 #define TR_CREATE_MODEL                 "Skapa modell"
-#define TR_FAVORITE_LABEL               "Favorites"
-#define TR_NEW_MODEL                    "New Model"
-#define TR_EDIT_LABELS                  "Edit Labels"
-#define TR_ENTER_LABEL                  "Enter Label"
-#define TR_LABEL                        "Label"
-#define TR_LABELS                       "Labels"
-#define TR_CURRENT_MODEL                "Current"
-#define TR_NEW                          "New"
-#define TR_NEW_LABEL                    "New Label"
-#define TR_RENAME_LABEL                 "Rename Label"
-#define TR_DELETE_LABEL                 "Delete Label"
+#define TR_FAVORITE_LABEL               "Favoriter"
+#define TR_NEW_MODEL                    "Ny modell"
+#define TR_EDIT_LABELS                  "Redigera etiketter"
+#define TR_ENTER_LABEL                  "Ange etikett"
+#define TR_LABEL                        "Etikett"
+#define TR_LABELS                       "Etiketter"
+#define TR_CURRENT_MODEL                "Nuvarande"
+#define TR_NEW                          "Ny"
+#define TR_NEW_LABEL                    "Ny etikett"
+#define TR_RENAME_LABEL                 "Byt namn på etikett"
+#define TR_DELETE_LABEL                 "Radera etikett"
 #define TR_DUPLICATE_MODEL              "Duplicera modell"
 #define TR_COPY_MODEL                   "Kopiera modell"
 #define TR_MOVE_MODEL                   "Flytta modell"
 #define TR_BACKUP_MODEL                 "Modellbackup"
 #define TR_DELETE_MODEL                 "Ta bort modell"
 #define TR_RESTORE_MODEL                "Återställ modell"
 #define TR_DELETE_ERROR                 "Delete error"
+#define TR_CAT_NOT_EMPTY                "Kategorin är ej tom"
```

Comment:
It looks like it's for deleting a non-empty category so can also go

Reply:
Gone!